### ISSUE TITLE
ScrollEvent.Position and ScrollEvent.AbsolutePosition is 0,0

### ISSUE THREAD

**Issue by pwiecz** [fyne-io/fyne#2199]
I create Scrolled(*fyne.ScrollEvent) method on my widget. It gets called when I scroll the mouse wheel, but the Position and AbsolutePosition fields of the ScrollEvent seem to be all zeroes.
I was expecting to find mouse position there.

OS: Linux (Ubuntu 20.04)
 Fyne version: v2.0.2

**Comment by Jacalz** [comment]
I see that you have chosen the bug template but discarded it. Can you please use the template (bug in this case) when opening issues? It is there to allow us to get enough, and the correct, information to be able to reproduce and help fix the issues.

**Comment by pwiecz** [comment]
Sorry about the.
If it helps I'll write the more complete report below.

### Describe the bug:
I create Scrolled(*fyne.ScrollEvent) method on my widget. It gets called when I scroll the mouse wheel, but the Position and AbsolutePosition fields of the ScrollEvent seem to be all zeroes.

### To Reproduce:
Steps to reproduce the behaviour:
1. Create a custom widget that implements Scrollable interface.
2.  Inside the Scrolled function log the ScrollEvent.Position.
3.  Add the widget to an app.
4. Run the app
2. Move mouse over the widget, and scroll the wheel (or use touchpad scrolling)
3. Look at the logs
4. Expected: { mouseX, mouseY}, got {0, 0}

### Screenshots:
Not - applicable

### Example code:
```go
package main

import (
	"fmt"

	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/widget"
)

type Widget struct {
	widget.BaseWidget
}

func (w *Widget) CreateRenderer() fyne.WidgetRenderer {
	return w
}
func (w *Widget) Scrolled(event *fyne.ScrollEvent) {
	fmt.Println(event.Position, event.AbsolutePosition)
}
func (w *Widget) Layout(size fyne.Size)        {}
func (w *Widget) MinSize() fyne.Size           { return fyne.Size{640, 480} }
func (w *Widget) Refresh()                     {}
func (w *Widget) Destroy()                     {}
func (w *Widget) Objects() []fyne.CanvasObject { return nil }
func main() {
	a := app.New()
	win := a.NewWindow("Test scroll")
	w := &Widget{}
	w.ExtendBaseWidget(w)
	win.SetContent(w)
	win.ShowAndRun()
}
```

## Device (please complete the following information):
 - **OS:** Linux
 - **Version:** 5.4.0-72
 - **Go version:** 1.16.2
 - **Fyne version:** 2.0.2

**Comment by Jacalz** [comment]
Thank you. Very much appreciated 🙂

**Comment by andydotxyz** [comment]
Fix in PR #2204, I hope we can land this for v2.0.3
